LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright 2017 Guardian News & Media Ltd
+Copyright 2019 Guardian News & Media Ltd
```

Comment:
```suggestion
Copyright 2017-2019 Guardian News & Media Ltd
```
technically I think we should cover all the years, or at least that's what I've had to do at more highly regulated places than here